### PR TITLE
Uf 8626 use multiple statuses for skolskjuts errands

### PR DESCRIPTION
**Pull request by josc04**

### CODE REVIEW

File: src/main/java/se/sundsvall/citizenchanges/service/ReminderService.java [hunk 1]
```diff
@@ -67,6 +67,7 @@ public BatchStatus runBatch(final int firstErrand, final int numOfErrands, final
 	}
 
 	public BatchStatus runBatch(final int firstErrand, final int numOfErrands, final boolean sendMessage, final List<String> oepErrands, final String sms, final String email) {
+		log.info("Batch job to send reminders is running...");
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/citizenchanges/service/ReminderService.java:70]: Follow Java naming conventions?

File: src/main/java/se/sundsvall/citizenchanges/service/ReminderService.java [hunk 2]
```diff
@@ -159,17 +164,16 @@ private void sendSmsIfRequired(final BatchContext batchContext, final OepErrandI
 	}
 
 	private void sendEmail(final boolean sendMessage, final String email, final OepErrandItem item, final String flowInstanceId) {
-
-		final var thisRecipient = Optional.ofNullable(email)
+		final var recipient = Optional.ofNullable(email)
 			.orElseGet(() -> item.getContactInfo().getEmailAddress());
 
 		final var reminderEmailSubject = LocalDate.now().getMonthValue() < 7 ? REMINDER_EMAIL_SUBJECT_SPRING + LocalDate.now().getYear() : REMINDER_EMAIL_SUBJECT_AUTUMN + LocalDate.now().plusYears(1).getYear();
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/citizenchanges/service/ReminderService.java:170]: Some magic numbers here. Constants?

File: src/main/java/se/sundsvall/citizenchanges/util/Constants.java [hunk 3]
```diff
@@ -49,14 +56,13 @@ public final class Constants {
...
-
 	public static final String PROTECTED_IDENTITY_INDICATOR = "Skyddad";
 
 	public static final int REMINDER_MAX_AGE = 16;
 
-	public static final String REMINDER_DATE_LIMIT_PATTERN_SPRING = "-08-31"; //The current year is prepended to this string (2022-08-31 etc.) to form the date limit.
+	public static final String REMINDER_DATE_LIMIT_PATTERN_SPRING = "-08-31"; // The current year is prepended to this string (2022-08-31 etc.) to form the date limit.
 
-	public static final String REMINDER_DATE_LIMIT_PATTERN_AUTUMN = "-01-31"; //The next year is prepended to this string (2024-01-31 etc.) to form the date limit.
+	public static final String REMINDER_DATE_LIMIT_PATTERN_AUTUMN = "-01-31"; // The next year is prepended to this string (2024-01-31 etc.) to form the date limit.
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/citizenchanges/util/Constants.java:65]: This is not a pattern (as the name implies), it's a suffix.

Maybe change this to a real pattern instead? I.e. :
`public static final String REMINDER_DATE_LIMIT_PATTERN_AUTUMN = "%s-01-31";`

Then use it:
`REMINDER_DATE_LIMIT_PATTERN_AUTUMN.formatted(theYear);`

File: src/main/java/se/sundsvall/citizenchanges/service/RelocationCheckService.java [hunk 4]
```diff
@@ -139,18 +153,18 @@ private List<InvestigationItem> mapFlowInstance(final String flowInstanceId, fin
...
 						if (sameAddress) {
-							log.info("Errand {} : The minor has moved to the same address as a guardian. Skip separate investigation item.", item.getFlowInstanceId());
+							LOGGER.info("Errand {} : The minor has moved to the same address as a guardian. Skip separate investigation item.", item.getFlowInstanceId());
 						} else {
 							tempList.add(itemMapper.composeInvestigationItem(thisMove, item, familyType, minorIdentifier, false));
 						}
 					});
 			}
-			log.info("tempList size: {}", tempList.size());
+			LOGGER.info("tempList size: {}", tempList.size());
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/citizenchanges/service/RelocationCheckService.java:165]: This seems to be a debug log print to me? 
Is "info" the correct level here?

Comment (josc04) [reply src/main/java/se/sundsvall/citizenchanges/service/RelocationCheckService.java:165]: Not sure, templist seems to be used to hold oep-errands to investigate. I know to little about the service to change level here, but I will rename tempList to better name (in log message and code).

**Review by foobar-1337 — APPROVED** [review]

**Review by AndreSoderlund — APPROVED** [review]

**Pull request merged**